### DISCUSSION TITLE
Provide sorting options

### DISCUSSION THREAD

**Discussion by stdedos** [useful-forks/useful-forks.github.io#23] [Ideas]
Hello there! Thank you for a fantastic addition to Github!
It is making life so much easier than doing it with shell magic and parsing the DOM ...
With that being said:

I am mostly interested on the projects that have the most modifications. And I can imagine different people might want to try different things based on different criteria.

Exposing more of the sorting options would be a nice addition to this nifty tool!

**Comment by payne911** [comment]
> I am mostly interested on the projects that have the most modifications. And I can imagine different people might want to different things based on different criteria.

Fair point, and I have considered it myself as well. However, this would quickly grow rather complex, not to mention all the UI work (which is my weakness) that would need to be done to enable the additional logic.

Maybe a list of the settings you had in mind could help narrow down on the specific needs of the community ?

Moreover, if you have the time and motivation for it, feel absolutely free to contribute a PoC : I will gladly review your PR. :)

**Comment by stdedos** [comment]
I have promised (the few time that I spare) to other open sources already; but be sure I'll round-robin to this one.

I have anyway cloned the project locally 😀

**Comment by payne911** [comment]
I'm thinking `useful-forks` could at least provide customization for deciding how many commits should be used to filter out based on `ahead_by` and `behind_by`.

There are too many instances where some forks are displayed just because someone added a `merge` commit on the default branch (`master` or `main`), for example. The same goes for forks which are thousands of commits behind (although that doesn't necessarily mean the fork doesn't contain some interesting additional content).

**Comment by payne911** [comment]
(This used to be Issue #12, but has now been converted to a Discussion.)
